api/composer.json

```diff
@@ -70,8 +70,31 @@
         ],
         "post-update-cmd": [
             "@auto-scripts"
+        ],
+        "symfony": [
+            "php bin/console"
+        ],
+        "load-fixtures": [
+            "@symfony hautelook:fixtures:load --no-interaction --no-bundles"
```

Comment:
Can you try the following syntax please?:
```
"hautelook:fixtures:load --no-interaction --no-bundles": "symfony-cmd"
```

If it works, you can do the same with the others custom scripts, and remove the "symfony" section

Reply:
No that doensn't work:

```
[10:20:34] COil@computerl:/api-platform-demo/api$ docker-compose exec php composer run load-fixtures
> symfony-cmd
sh: symfony-cmd: not found
```